### ISSUE TITLE
Links in AI and Data Scientist Roadmap seem to be broken

### ISSUE THREAD

**Issue by kdmelon** [kamranahmedse/developer-roadmap#7683]
### What is the URL where the issue is happening

https://roadmap.sh/ai-data-scientist

### What browsers are you seeing the problem on?

Chrome

### Describe the Bug

Some or most links under Mathematics, Statistics, Exploratory Data Analysis, MLOps, and Deep Learning seem to be broken. A window pops up but quickly closes right after. It seems all the Coursera links are broken too.

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by omdhede** [comment]
@kdmelon Hey, can you specify which link is not working!? Because I checked the entire section and couldn't find any link that was broken.

**Comment by kamranahmedse** [comment]
It is probably some browser extension, can you please try in incognito or a different browser?

**Comment by kdmelon** [comment]
Hi, I tried [Mathematics for Machine Learning Specialization](https://imp.i384100.net/baqMYv), [Algebra and Differential Calculus for Data Science](https://imp.i384100.net/LX5M7M), [Introduction to Statistics](https://imp.i384100.net/3eRv4v). The links seemed to work only after I manually pasted them into my browser. I haven't been able to replicate the issue since then. It might just be a quirk with my browser.
